index.js

```diff
@@ -209,7 +209,7 @@ class ServerlessS3Sync {
       if (!s.bucketName || !s.localDir) {
         throw 'Invalid custom.s3Sync';
       }
-      const localDir = [servicePath, s.localDir].join('/');
+      const localDir = [servicePath, s.localDir].join('/').replace(/\/?$/, '/');
```

Comment:
I would suggest to use node.js' [path.join([...paths])](https://nodejs.org/api/path.html#path_path_join_paths), because it uses the platform-specific separator as delimiter (e.g. `/` on POSIX and either `\` or `/` on Windows). You can get rid of the regex then, because it also replaces multiple separators into a single instance:

![image](https://user-images.githubusercontent.com/793157/73161106-0f83a780-40eb-11ea-9c61-6e655edeb40f.png)

Reply:
Thanks for the hint.  Evidently my JS skills are far more out of date that I realised.